### DISCUSSION TITLE
Extension being cut off

### DISCUSSION THREAD

**Discussion by rkamarowski** [wixtoolset/issues#7722] [Questions] [answered]
I have an executable named: Photography.UI.exe

Windows installer is cutting off the exe, so the file name is: Photography.UI

Am I doing something wrong?
Bob K.

**Comment by robmen** [comment]
> Am I doing something wrong?

Hard to say. You don't say what you are doing. Try editing your question to include three parts:

1. Repro steps (precise steps and any code necessary to reproduce the behavior).
2. What behavior you'd like to see.
3. What behavior you actually see.

**Comment by rkamarowski** [comment, marked as answer]
Wix 4
VS 2022
Application project and Installer project are in the same solution.

I'm attempting to get the exe file from my project and with the install place it in Program Files\Manufacturer\ProductName\exe file
There are no build errors. Install is working except for the exe extension. What shows is Photography.UI
FilePath = D:\SoftwareMVVMTCH\EES\Photography.UI\bin\x64\Debug\net7.0-windows\\
Code snippet attached.

```
<Component 
		Id='MainExecutable' 
		Guid='{C51544FD-9F4B-40FC-BAF4-3C8D0DFF569B}'
		Bitness ='always64'>
		<File 
				Id='PhotographyEXE' 
				Name='Photography.UI.EXE' 
				DiskId='1' 
				Source='$(FilesPath)Photography.UI.exe' 
				KeyPath='yes'>
```
I'm currently going through your Dojo Deployment series to get a handle on version 4.

**Reply by robmen** [reply]
What does the log file show when installing the file? Also, quick check: Did you turn on the File Explorer setting to show file extensions?

**Comment by rkamarowski** [comment]
I'm an idiot...
Show extension wasn't on. Thanks for the quick reply.
